crm-project/src/main/java/CRM/service/CommentService.java

```diff
@@ -50,12 +59,18 @@ public List<SharedContent> getAllInItem(long itemId) {
         return null;
     }
 
-    public Comment create(CommentRequest comment){
-        // I don't think there are any validations needed to be here, since we
-        // don't have any unique or primary values in an item.
+    public Comment create(CommentRequest commentRequest) throws AccountNotFoundException {
+        User user;
```

Comment:
move it right above the try block as it is connected to it. 
If Validations.doesIdExists throws exception, you created a user in memory for no reason